utils/spdx/src/main/kotlin/SpdxExpression.kt

```diff
@@ -638,3 +633,23 @@ data class SpdxLicenseReferenceExpression(
 
     override fun getLicenseUrl(): String? = null
 }
+
+fun Collection<SpdxExpression>.and(): SpdxExpression = concat(SpdxOperator.AND)
```

Comment:
Can you please add docs for the new functions?

Reply:
Actually, some test cases would also be nice.

Reply:
Can I make this in a folllow-up PR, so that this can go into the release you plan to make today?

Reply:
Yes, the would be fine with me.

Reply:
@mnonnenmacher here you go: 
- https://github.com/oss-review-toolkit/ort/pull/9917
- https://github.com/oss-review-toolkit/ort/pull/9915